test/functional/tests/conftest.py

```diff
@@ -178,11 +182,15 @@ def base_prepare(item):
 
         if get_force_param(item) and not TestRun.usr.already_updated and not uninstall_cas:
             installer.rsync_opencas_sources()
-            installer.reinstall_opencas()
+            installer.reinstall_opencas(
```

Comment:
`installer.rsync_opencas_sources()` could be renamed to `installer.rsync_opencas()` and expect `copy_rpm` and `rpm_path` arguments